docs/rules/self-closing-comp.md

```diff
@@ -17,5 +17,56 @@ var contentContainer = <div className="content"></div>;
 
 var HelloJohn = <Hello name="John" />;
 
-var Profile = <Hello name="John"><img src="picture.png" /></Hello>; 
+var Profile = <Hello name="John"><img src="picture.png" /></Hello>;
+```
+
+## Rule Options
+
+The rule can take one argument to select types of tags, which should be self-closed when this is possible. By default only custom components tags should be self-closed.
+
+```js
+...
+"self-closing-comp": [<enabled>, {
+  "component" <boolean> || true,
```

Comment:
`|| true` means it will never be false, so i don't think this is good notation. I'd just provide the default object:
```js
"self-closing-comp": ["error", {
  "component": true,
  "html": false
}]
```

I think it's pretty self-apparent that you can override them with your own booleans